src/openforms/conf/base.py

```diff
@@ -232,7 +232,7 @@
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
     "axes.middleware.AxesMiddleware",
     "csp.contrib.rate_limiting.RateLimitedCSPMiddleware",
-    "csp.middleware.CSPMiddleware",
+    "openforms.utils.middleware.UpdateCSPMiddleware",
```

Comment:
wouldn't these have to be swapped? since the responses are applied from bottom to top, our custom updates would have to be in addition to the policy set from the settings by the `RateLimitedCSPMiddleware`?